### ISSUE TITLE
Screen reader not announcing icons and buttons on address books bar in contacts

### ISSUE THREAD

**Issue by Unwrap-at** [mustang-im/mustang#294]
Screen reader users rely on labels to understand the purpose of an element. 
For example, the icons and buttons on the left side bar are not announced by the screen reader, leaving
blind or visually impaired users unaware of their function.

![image](https://github.com/user-attachments/assets/8cd703f9-e4fc-4190-88c8-4cb16af294e9)
